.github/workflows/scripts/helm-weekly-release.sh

```diff
@@ -42,6 +42,17 @@ increment_semver() {
   echo $(local IFS=$delimiter ; echo "${array[*]}")
```

Comment:
```suggestion
  array[$2]="0"
  echo $(local IFS=$delimiter ; echo "${array[*]}")
```
Alternative to my previous comment, we also can just zero the patch number above this line. It slightly violates the single responsibility of a single function should have, but since it is a very small function anyway, maybe that should be fine. If we do this, of course, we also should update the function name to something like `increment_minor_and_zeroes_patch_semver`. It is a bit mouthful, indeed.

Reply:
I kept the two functions if you don't mind. I like that it's more explicit how the version is determined. But I changed one of them to set the patch to zero instead of setting any arbitrary segment to any arbitrary value.